modules/simple-payments/paypal-express-checkout.js

```diff
@@ -161,14 +161,16 @@ var PaypalExpressCheckout = {
 								return reject( new Error( 'server_error' ) );
 							}
 
+							PaypalExpressCheckout.spayOrderId = paymentResponse.order_id;
```

Comment:
> PaypalExpressCheckout object is global for all payment buttons on a site.
> Order id should be local to the transaction itself or at very lest the button.

Theoretically, the buyer shouldn't make two or more payments at the same time. The first blocker is the visual mask in the website.

<img src="https://user-images.githubusercontent.com/77539/29458867-8ac56a72-83f7-11e7-91fa-e1e9cd50354f.png" width="400px" />

I removed the overlay trying to open more than one Paypal Popup. BTW, PayPal sometimes opens its dialog using HTML instead of the browser native.

<img src="https://user-images.githubusercontent.com/77539/29458901-b403c05a-83f7-11e7-9fab-300ef4bfc71e.png" width="400px" />

I got three PP Dialogs :-o 

<img src="https://user-images.githubusercontent.com/77539/29458957-02fd6968-83f8-11e7-9259-fcbc8606ea38.png" width="400px" />

... it was ok :-p

<img src="https://user-images.githubusercontent.com/77539/29459005-3f10cf62-83f8-11e7-9815-17365da091ff.png" width="400px" />

> To make it a local scope of the button, it should be a local variable in the renderButton function.

👍 

> ALSO we could make a global array that maps paymentResponse.id to paymentResponse.order_id. I think that would be safest.

👍